src/Controls/src/Core/Platform/GestureManager/GestureManager.iOS.cs

```diff
@@ -77,24 +88,22 @@ public void Dispose()
 					tapGestureRecognizer.PropertyChanged -= OnTapGestureRecognizerPropertyChanged;
 				}
 
-				if (_platformView != null)
-					_platformView.RemoveGestureRecognizer(kvp.Value);
+				if (PlatformView != null)
+					PlatformView.RemoveGestureRecognizer(kvp.Value);
 				kvp.Value.ShouldReceiveTouch = null;
 				kvp.Value.Dispose();
 			}
 
-			if (_platformView != null && OperatingSystem.IsIOSVersionAtLeast(11))
+			if (PlatformView != null && OperatingSystem.IsIOSVersionAtLeast(11))
 			{
 				foreach (IUIInteraction interaction in _interactions)
 				{
-					_platformView.RemoveInteraction(interaction);
+					PlatformView.RemoveInteraction(interaction);
 				}
 			}
 			_gestureRecognizers.Clear();
 
 			Disconnect();
-
-			_platformView = null;
```

Comment:
Should we still set the weakreference to null here? or initialize it with a null target? So we stay consistent?

Reply:
done!